modules/ROOT/pages/composite-databases/queries.adoc

```diff
@@ -223,7 +223,7 @@ CALL {
   MATCH (movie:Movie)
   RETURN movie
 }
-RETURN movie
+RETURN [p=(movie)-->() | p] AS paths
```

Comment:
So is the intention to give an invalid query as the example here? Perhaps we could give a valid query as the first example here, and then for the `MATCH (n)`, `CREATE (:X)`, `WITH [p=()--() | p] AS paths` below, we put full, _invalid_ queries using those examples?

Reply:
@smietana, could you be able to address this comment before I can incorporate it in my PR?